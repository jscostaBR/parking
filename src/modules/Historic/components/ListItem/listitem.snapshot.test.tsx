import * as React from 'react'
import { ListItem } from '.'
import renderer from 'react-test-renderer'

describe('render ListItem', () => {
  it('render correctly', () => {
    const tree = renderer.create(<ListItem time="1 hora 12 min" payment={true} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
