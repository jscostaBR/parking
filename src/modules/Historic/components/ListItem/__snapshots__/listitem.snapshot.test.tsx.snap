// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ListItem render correctly 1`] = `
<div
  className="sc-bczRLJ kIPHBJ"
>
  <section>
    <p>
      Tempo 
      total
    </p>
    <strong>
      1 hora 12 min
    </strong>
  </section>
  <section>
    <p>
      Pagamento
    </p>
    <strong>
      Pago
    </strong>
  </section>
</div>
`;
